plugins/auth-backend/api-report.md

```diff
@@ -47,9 +47,12 @@ export type AtlassianProviderOptions = {
   };
 };
 
+// Warning: (ae-forgotten-export) The symbol "AuthContext" needs to be exported by the entry point index.d.ts
+//
 // @public
 export type AuthHandler<AuthResult> = (
   input: AuthResult,
+  ctx?: AuthContext,
```

Comment:
Makes sense 👍  I think we'd really want it to be required, but can do that in a followup change. Best make sure it's exported from the package though, and I think perhaps a slightly less generic name, maybe `AuthResolverContext`?

Reply:
Adjusted 👍